source/crud/tests/unified/create-null-ids.yml

```diff
@@ -0,0 +1,169 @@
+description: create-null-ids
+schemaVersion: '1.0'
+
+createEntities:
+  - client:
+      id: client
+      observeEvents:
+        - commandStartedEvent
```

Comment:
```suggestion
```

Looks like you got rid of APM assertions.

Reply:
I figured the result of count is the source of truth now since we are able to use a BSON type query avoiding any comparison looseness surrounding null/undefined. But happy to add it back, just saying it was intentional to remove.

Reply:
`countDocuments` seems sufficient. APM assertions don't help if we have to worry about how some drivers represent null/undefined values.